programs/validator-history/src/state.rs

```diff
@@ -64,7 +135,9 @@ pub struct ValidatorHistoryEntry {
     pub vote_account_last_update_slot: u64,
     // MEV earned, stored as 1/100th SOL. mev_earned = 100 means 1.00 SOL earned
     pub mev_earned: u32,
-    pub padding1: [u8; 84],
+    /// The enum mapping of the Validator's Tip Distribution Account's merkle root upload authority
+    pub merkle_root_upload_authority: MerkleRootUploadAuthority,
```

Comment:
Should we actually updated `padding0` to be the `MerkleRootUploadAuthority` since it's a u8?